test/integration-legacy/selenium-helpers.js

```diff
@@ -0,0 +1,168 @@
+const webdriver = require('selenium-webdriver');
```

Comment:
Why is this categorized as legacy?

Reply:
There are currently two copies of selenium-helpers, one in the legacy and one in the current integration tests.  They're currently the same, but we may have different needs when using Jest so I allowed them to be separate.  I suspect these files will diverge as work is done on the integration tests.